src/popup/components/TransferSendForm.vue

```diff
@@ -85,7 +85,7 @@
     </div>
 
     <InputAmount
-      v-model="formModel.amount"
+      v-model.number="formModel.amount"
```

Comment:
Even if you want to stick to this approach there are ~10 places where `InputAmount` is used, and where this line should be used in this case.

Reply:
updated for other components too